apps/01PigLatin.js

```diff
@@ -3,86 +3,86 @@
 var assert = require('assert');
 var readline = require('readline');
 var rl = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout
+  input: process.stdin,
+  output: process.stdout
 });
 
 
 function pigLatin(word) {
-    word = word.toLowerCase();
+  word = word.toLowerCase();
 
-    var firstLetter = word[0];
+  var firstLetter = word[0];
 
-    var vowelIndex = -1; // Set it to assume there are no vowels
+  var vowelIndex = -1; // Set it to assume there are no vowels
 
-    if ((word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex) || vowelIndex === -1) {
-        vowelIndex = word.indexOf('a');
-    }
+  if ((word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('a');
+  }
 
-    if ((word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex) || vowelIndex === -1) {
-        vowelIndex = word.indexOf('e');
-    }
+  if ((word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('e');
+  }
 
-    if ((word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex) || vowelIndex === -1) {
-        vowelIndex = word.indexOf('i');
-    }
+  if ((word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('i');
+  }
 
-    if ((word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex) || vowelIndex === -1) {
-        vowelIndex = word.indexOf('o');
-    }
+  if ((word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('o');
+  }
 
-    if ((word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex) || vowelIndex === -1) {
-        vowelIndex = word.indexOf('u');
-    }
+  if ((word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('u');
+  }
 
-    if ((word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex) || vowelIndex === -1) {
-        vowelIndex = word.indexOf('y');
-    }
+  if ((word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex) || vowelIndex === -1) {
+    vowelIndex = word.indexOf('y');
+  }
 
-    var firstPart = word.slice(0, vowelIndex);
+  var firstPart = word.slice(0, vowelIndex);
 
-    var restWord = word.slice(vowelIndex, word.length);
+  var restWord = word.slice(vowelIndex, word.length);
 
-    if (vowelIndex === 0) {
-        return word + 'yay';
-    } else {
-        return restWord + firstPart + 'ay';
-    }
+  if (vowelIndex === 0) {
```

Comment:
I had made the mistake of leaving a "return" command before this and couldnt get it to work for a while, glad to see you didnt make that same error. it was frustrating.